x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -71,6 +78,12 @@ const resolverTreeResponse = (state: DataState): NewResolverTree | undefined =>
   return state.tree?.lastResponse?.successful ? state.tree?.lastResponse.result : undefined;
 };
 
+const lastResponseIndices = (state: DataState): string[] | undefined => {
```

Comment:
I'm probably missing something but could we get away without needing to look at the last response? Could we just use the current indices all the time?

Reply:
I think this helps us maintain the original indices which were used to successfully get data back. These are the "current" indices technically as far as I can see

Reply:
@jonathan-buttner The reason for doing this is because all of these event api requests are inherently tied to the tree the user is currently viewing, if a separate set of indices is passed to resolver as the middlewares are running, that would be a bug. Should not happen currently, as changing the indices with sourcerer forces the store to be recreated, but that won't necessarily always be the case. Added the sequence of actions that show the need for this here: https://github.com/elastic/kibana/pull/90727/files#diff-0a7459663dc51fee4faacb117efa940c3a4f5dfdfa1abc44ea873507e8de80aaR706